### PR TITLE
added workflow changes

### PR DESCRIPTION
**Pull request by BrandonMurnane**

### CODE REVIEW

**Comment by kriscfoster** [comment]
@BrandonMurnane pull from master and push again, you're branch is out of date I think

File: refocus-bdk-server.js [hunk 1]
```diff
@@ -175,7 +175,12 @@ function genericPost(route, obj, proxy, apiToken){
 
 module.exports = (config) => {
   const SERVER = config.refocusUrl;
-  const TOKEN = config.token;
+  let TOKEN;
+  if (!NEW_TOKEN_WORKFLOW) {
+    TOKEN = config.token;
+  }
+  const BOT_INSTALL_TOKEN = config.token;
+  const SOCKET_TOKEN;
```
Comment (kriscfoster) [inline refocus-bdk-server.js]: @BrandonMurnane I think this should be let

**Review by kriscfoster — COMMENTED** [review]
Just one thing

**Review by tausifmuzaffar — APPROVED** [review]

File: refocus-bdk-server.js [hunk 2]
```diff
@@ -426,6 +431,10 @@ module.exports = (config) => {
           } else {
             // Need to save this after install
             logger.info('Socket Authorization Token: ' + res.body.token);
+            SOCKET_TOKEN = res.body.token;
+            if (NEW_TOKEN_WORKFLOW) {
```
Comment (kriscfoster) [inline refocus-bdk-server.js]: Is this a toggle, it's not defined anywhere?

File: refocus-bdk-server.js [hunk 3]
```diff
@@ -499,6 +508,10 @@ module.exports = (config) => {
 
             return reject(err || !ok);
           }
+          if (NEW_TOKEN_WORKFLOW) {
```
Comment (kriscfoster) [inline refocus-bdk-server.js]: @BrandonMurnane this is undefined

**Review by kriscfoster — APPROVED** [review]

**Pull request merged**